### PR TITLE
[RandomTranslation] Supporting Segmentation Masks

### PR DESCRIPTION
**Pull request by IMvision12**
Related #1992

What tests i need to add for this?

@ianstenbit 

Capture

### CODE REVIEW

**Comment by cosmo3769** [comment]
Hi @IMvision12 Could you please update your description from `Fixes` to `Related`. This is because if this PR will be merged then automatically #1992 will be closed.

**Review by ianstenbit — APPROVED** [review]
Thanks for the PR -- just one small fix but otherwise it looks good!

File: keras_cv/layers/preprocessing/random_translation.py [hunk 1]
```diff
@@ -201,6 +201,34 @@ def augment_images(self, images, transformations, **kwargs):
...
+        height_translations = height_translations * img_hd
+        width_translations = width_translations * img_wd
+        translations = tf.cast(
+            tf.concat([width_translations, height_translations], axis=1),
+            dtype=tf.float32,
+        )
+        output = preprocessing_utils.transform(
+            segmentation_masks,
+            preprocessing_utils.get_translation_matrix(translations),
+            interpolation=self.interpolation,
```
[the remark this answers is not in the document]
Comment (IMvision12) [reply keras_cv/layers/preprocessing/random_translation.py]: Done!

**Comment by ianstenbit** [comment]
@IMvision12 please take a look at the code format error :)

**Comment by IMvision12** [comment]
Done @ianstenbit

**Review by ianstenbit — APPROVED** [review]
thank you!

**Pull request merged**